### DISCUSSION TITLE
Tengo problemas, estoy repitiendo el curso y me aparecen las notas de antes, no me deja hacer nada

### DISCUSSION THREAD

**Discussion by mateomunozg** [rramosp/ai4eng.v1#324] [00 - General]
quedo atento 
![image](https://github.com/user-attachments/assets/1cf52f15-0f09-4ea2-9d61-b7ac7000ecf2)

**Comment by Felipe-RA** [comment]
@mateomunozg 

Hay un post #212  especifico para estos casos.

Sigue las instrucciones que hay alli
